tst/Pester.RSpec.ts.ps1

```diff
@@ -982,6 +982,130 @@ i -PassThru:$PassThru {
                 }
             }
         }
+
+        t "Undefined parameters with default values are automatically added to container Data" {
+            try {
+                $sb = {
+                    param (
```

Comment:
Maybe do a quick rewrite of the tests to keep simple integration-tests with `Invoke-Pester` here while checking the logic and different parametertypes etc. in `Pester.Runtime.ts.ps1` with `Invoke-Test` + `New-BlockContainerObject`?